getting-started/glossary.md

```diff
@@ -1,51 +1,93 @@
+## GCC
+The Government Commercial Cloud Service brings the modern innovations and capabilities of commercial cloud computing platforms to less sensitive Government systems. These leading ICT capabilities are augmented by robust cybersecurity measures and systems to protect the data that resides on commercial cloud platforms.
 
-## Content Disarm and Reconstruct \(CDR\)
+
+## HTTPS
+Hypertext Transfer Protocol Secure is an extension of the Hypertext Transfer Protocol. It is used for secure communication over a computer network, and is widely used on the Internet.
+
+## IM8
+Instruction Manual 8. A management tool used in the Government to safeguard ICT & SS (Infocomm Technology and Smart Systems) assets and guides all Government agencies in adopting ICT & SS to enable their digital transformation.
 
 
 ## JWT
-JSON Web Token (JWT) is an open standard (RFC 7519) that defines a compact and self-contained way for securely transmitting information between parties as a JSON object.
+JSON Web Token is an open standard (RFC 7519) that defines a compact and self-contained way for securely transmitting information between parties as a JSON object.
+
+## KMS
+Key Management Service by AWS helps you create and manage cryptographic keys and control their use across a wide range of AWS services and in your applications.
+
+## Malware Scanning
+The process of detecting malware in a file.
 
 ## OpenAPI Specifications
 The OpenAPI Specification (OAS) defines a standard, language-agnostic interface to RESTful APIs which allows both humans and computers to discover and understand the capabilities of the service without access to source code, documentation, or through network traffic inspection.
 
 ## Payload
 The body of an API request or response message.
 
-## Project Id
+## PGP
+Pretty Good Privacy is an encryption program that provides cryptographic privacy and authentication for data communication. 
 
 ## REST API
 A REST API is an API that conforms to the design principles of the REST, or representational state transfer architectural style.
 
 ## Sandbox
 CFT sandbox environment is a secure environment to try out CFT APIs. The sandbox supports file transfers in the internet zone between systems.
 
+## SG Tech Stack (SGTS)
+SGTS consists of a series of common applications and platforms built by the Government that helps agencies build compliant applications easily.
+
+## Amazon S3
+Amazon Simple Storage Service is an object storage service that offers industry-leading scalability, data availability, security, and performance.
+
+## SSE
+Streaming SIMD Extensions is a processor technology that enables single instruction multiple data. It enables the instructions to handle multiple data elements.
+
+## TechPass
+A Single Sign-On, Identity & Access Management solution for developer services in SGTS, enabling users to transition seamlessly between services, while providing downstream services a seamless onboarding and consumption experience.
```

Comment:
"... to transit seamlessly between..." 

"downstream services for a seamless ... "